coverage/pom.xml

```diff
@@ -21,11 +21,11 @@
   <parent>
     <groupId>com.google.cloud.bigdataoss</groupId>
     <artifactId>bigdataoss-parent</artifactId>
-    <version>3.0.0-SNAPSHOT</version>
```

Comment:
why is this not updated by the release automation?

Reply:
It did. Check commitId: [8d7c5cd](https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1091/commits/8d7c5cd35f43067fe6ac4de7867e8a7dbb273a09)

Changes I have to rollback were specifically in commit: [13877a] https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1091/commits/b03bea4e0c36270f8d6cb60f10db1acf5013877a

As our automation script checks for version: `3.0.0` and replaced it with `3.0.1-SNAPSHOT`. It caused a wrong package update. This collision should not occur that frequently now.